### PR TITLE
cleaning up & joining daily and monthly

### PR DESCRIPTION
**Pull request by chloemurphy98**

### CODE REVIEW

File: dags/sql/reports/rep__shopify_partner_daily_summary.sql [hunk 1]
```diff
@@ -3,79 +3,96 @@ DROP MATERIALIZED VIEW IF EXISTS {{ schema }}.rep__shopify_partner_daily_summary
...
+    WITH base_orders AS (
+   SELECT
+       CASE
+           WHEN (order_type = 'harper_try' OR harper_product = 'harper_try') THEN 'harper_try'
+           WHEN (tags LIKE '%%harper%%' OR payment_gateway_names = 'Harper Payments' OR co.original_order_name IS NOT NULL) THEN 'harper_concierge'
+           ELSE NULL
+       END AS harper__product,
+       po.name,
+       CASE
+           WHEN LOWER(po.shipping_address__city) = 'london' THEN 'London'
```
Comment (martyzz1) [inline dags/sql/reports/rep__shopify_partner_daily_summary.sql:15]: still think this should be in the data import... not calculated everytime we run a query

File: dags/sql/reports/rep__shopify_partner_daily_summary.sql [hunk 2]
```diff
@@ -3,79 +3,96 @@ DROP MATERIALIZED VIEW IF EXISTS {{ schema }}.rep__shopify_partner_daily_summary
...
+       po.created_at,
+       po.partner__name,
+       po.partner__reference,
+       po.items_ordered,
+       po.items_returned,
+       po.value_ordered,
+       po.value_returned,
+       po.cancelled_at,
+       CASE
+           WHEN tags LIKE '%%harper%%' OR payment_gateway_names = 'Harper Payments' OR co.order_name IS NOT NULL THEN 'Harper'
```
Comment (martyzz1) [inline dags/sql/reports/rep__shopify_partner_daily_summary.sql:27]: this absolutely should be in the Data import...

File: dags/sql/reports/rep__shopify_partner_monthly_summary.sql [hunk 3]
```diff
@@ -3,78 +3,96 @@ DROP MATERIALIZED VIEW IF EXISTS {{ schema }}.rep__shopify_partner_monthly_summa
...
-    FROM
-        {{ schema }}.shopify_partner_orders po
-    LEFT JOIN {{ schema }}.clean__order__summary co
-    ON po.name = co.original_order_name
-            )
+   WITH base_orders AS (
+  SELECT
+      CASE
+          WHEN (order_type = 'harper_try' OR harper_product = 'harper_try') THEN 'harper_try'
+          WHEN (tags LIKE '%%harper%%' OR payment_gateway_names = 'Harper Payments' OR co.original_order_name IS NOT NULL) THEN 'harper_concierge'
```
Comment (martyzz1) [inline dags/sql/reports/rep__shopify_partner_monthly_summary.sql:10]: this is likely to be wrong... tags LIKE '%%harper%%' will match ALL harper orders....

**Review by chloemurphy98 — COMMENTED** [review]
1. Clean shopify partner view
2. shopify_partner_daily_summary -> view 
3. shopify_partner_monthly_summary -> view 
4. delete unnecessary view
5. correct logic which flags harper order type

**Pull request merged**